projects/igniteui-angular/src/lib/dialog/dialog.component.spec.ts

```diff
@@ -360,6 +361,71 @@ describe('Dialog', () => {
         expect(childDialog.isOpen).toEqual(false);
     }));
 
+    it('Should chenaged closeOnOutsideSelect property while the dialog is open', fakeAsync(() => {
```

Comment:
extend the test, or add another one, to check in opposite direction if `closeOnOutsideSelect` is changed from `false` to `true` while the dialog is open.